modules/cco-short-term-creds-auth-flow-gcp.adoc

```diff
@@ -6,6 +6,8 @@
 [id="cco-short-term-creds-auth-flow-gcp_{context}"]
 = GCP Workload Identity authentication process
 
+Requests for new and refreshed credentials are automated by using an appropriately configured OpenID Connect (OIDC) identity provider, combined with IAM service accounts. {product-title} signs service account tokens that are trusted by GCP, and can be projected into a pod and used for authentication. Tokens are refreshed after one hour by default.
```

Comment:
The same suggestions apply here.

You mention "by default" here but not in the AWS version. Is that intentional, or is one way correct for both?

Reply:
I do not believe either can be configured by the user here, so probably it should just be "Tokens are refreshed after one hour." @jianping-shu does that make sense to you for GCP?

Reply:
@jeana-redhat Sorry for missing this question. I think the default duration is 1 hour for token in both AWS and GCP, CCO doesn't configure the duration explicitly and just use the default.
Here is the GCP doc
https://cloud.google.com/docs/authentication/token-types#at-lifetime

Reply:
No problem! thanks for the confirmation